templates/quickstart-bitbucket-dc-with-vpc.template.yaml

```diff
@@ -598,6 +605,7 @@ Resources:
         DeploymentAutomationBranch: !Ref 'DeploymentAutomationBranch'
         DeploymentAutomationPlaybook: !Ref 'DeploymentAutomationPlaybook'
         DeploymentAutomationKeyName: !Ref 'DeploymentAutomationKeyName'
+        DeployRepositoryCustomParams: !Ref 'DeployRepositoryCustomParams'
```

Comment:
Should be 'DeploymentAutomationCustomParams'

Reply:
https://server-syd-bamboo.internal.atlassian.com/browse/DCD-AWSBITBUCKET19-LINT-2